HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2PixelTracks_cfi.py

```diff
@@ -1,20 +1,23 @@
 import FWCore.ParameterSet.Config as cms
 
-hltPhase2PixelTracks = cms.EDProducer("PixelTrackProducer",
-    Cleaner = cms.string('pixelTrackCleanerBySharedHits'),
-    Filter = cms.InputTag("hltPhase2PixelTrackFilterByKinematics"),
-    Fitter = cms.InputTag("hltPhase2PixelFitterByHelixProjections"),
-    SeedingHitSets = cms.InputTag("hltPhase2PixelTracksHitSeeds"),
-    mightGet = cms.optional.untracked.vstring,
-    passLabel = cms.string('hltPhase2PixelTracks')
-)
-
-from Configuration.ProcessModifiers.alpaka_cff import alpaka
-_hltPhase2PixelTracks = cms.EDProducer("PixelTrackProducerFromSoAAlpakaPhase2",
+hltPhase2PixelTracks = cms.EDProducer("PixelTrackProducerFromSoAAlpakaPhase2",
```

Comment:
Is there a reason to remove completely what has been used as the baseline for HLT tracking so far?
The change, as is, would make it impossible to resort the previous configuration, even if just for comparisons.
Why not just using a procModifier, like it's normally done when a change is introduced to allow for testing at a larger scale?
@cms-sw/tracking-pog-l2